internal/pkg/template/templates/task/cf.yml

```diff
@@ -69,8 +62,11 @@ Resources:
           Environment:{{range $name, $value := .EnvVars}}
           - Name: {{$name}}
             Value: {{$value}}{{end}}{{end}}
-          {{- if .Secrets}}
-          Secrets:{{range $name, $valueFrom := .Secrets}}
+          {{- if .SSMParamSecrets}}
+          Secrets:{{range $name, $valueFrom := .SSMParamSecrets}}
```

Comment:
Right make sense, I have modified if condition to - `{{- if or .SSMParamSecrets .SecretsManagerSecrets}}` So both the types of secrets will be added in `secrets:`

Reply:
ohh right, I have modified if condition with `{{- if or .SSMParamSecrets .SecretsManagerSecrets}}` so that `Secrets:` will have both SSMParamSecrets and SecretsManagerSecrets